src/resources/auth/auth.router.ts

```diff
@@ -8,4 +8,8 @@ router.route('/signup').post(validateSignUp(), authController.signup)
 
 router.route('/signin').post(validateSignIn(), authController.signin)
 
+router.route('/password/forget').post(authController.forgetPassword)
+
+router.route('/password/reset/:resetToken').post(authController.resetPassword)
+
```

Comment:
Add validation to /forget and /reset route. And let's discuss about those routes later. It is a little bit long url /auth/password/forget